modules/nw-endpoint-route53.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="nw-endpoint-route53_{context}"]
+= Endpoint configuration for route53
+
+If you install in either GovCloud region, the Ingress Operator uses `us-gov-west-1` region for Route53 and tagging API clients.
+
+The Ingress Operator uses `https://tagging.us-gov-west-1.amazonaws.com` as the tagging API endpoint if a tagging custom endpoint is configured that includes the string 'us-gov-east-1'.
+
+[IMPORTANT]
+====
+Private, disconnected installs are not supported for AWS GovCloud when installing in the `us-gov-east-1` region.
+====
+
+.Example `route53` configuration
+[source,yaml]
+----
+platform:
+  aws:
+    region: us-gov-west-1
+    serviceEndpoints:
+    - name: ec2
+       url: https://ec2.us-gov-west-1.amazonaws.com
+    - name: elasticloadbalancing
+       url: https://elasticloadbalancing.us-gov-west-1.amazonaws.com
+    - name: s3
+       url: https://s3.us-east-1.amazonaws.com
+    - name: tagging
+       url: https://tagging.us-east-1.amazonaws.com
```

Comment:
the url should be `https://tagging.us-gov-west-1.amazonaws.com`

Reply:
Thank you, @lihongan! I updated that URL and changed the one for s3 to `https://route53.us-gov.amazonaws.com`.